crates/rolldown/src/ecmascript/format/esm.rs

```diff
@@ -138,7 +144,11 @@ fn render_esm_chunk_imports(ctx: &GenerateContext<'_>) -> String {
         }
       }
       RenderImportDeclarationSpecifier::ImportStarSpecifier(alias) => {
-        s.push_str(&format!("import * as {alias} from \"{path}\";\n",));
+        s.push_str("import * as ");
```

Comment:
Maybe a helper method like

```rust
trait StringExt {
  fn push_strs((&mut self, strs: &[&str]) {
    let size_hint = ...;
    self.reserve_addiational();
    // push_str
  }  
}
}
```

Macro would be better for this due to this will create a temporary array.

Reply:
Let's do it in a separate pr